[KOGITO-7705 Updating grpc documentation (#126)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

In the same GitHub repository as the example application, there is a link:{grpc_example_url}/serverless-workflow-greeting-server-rpc-quarkus[Maven project] which provides a simple implementation of the `Greeter` service. By default, this gRPC server runs on port `50051`. Therefore, the same port is used in the `application.properties` file.

=== Default enum values

> grpc link:{grpc_enum_url}[specification] requires enumeration types to have a default value. Default value is not included in the server response payload. Therefore it is recommended to use as default a meaningless value like UNKNOWN. 

Reviewer: ```suggestion
gRPC link:{grpc_enum_url}[specification] requires enumeration types to have a default value. The default value is not included in the server response payload. Therefore, use an empty value such as `UNKNOWN` as default.
```

> grpc link:{grpc_enum_url}[specification] requires enumeration types to have a 0 default value. Default value is not included in the server response payload. Therefore it is recommended to use as default a meaningless value like UNKNOWN. 

Reviewer 2: what do you mean by have a 0 default value?
Isn't better just say that "requires enum types to have no default value" ?

Author: The fact that the default value 0 is not really relevant so I will remove that for clarity

> If, for any reason, your default value is semantically valid and you want it to be included in the Workflow model, you must set `kogito.grpc.enum.includeDefault` to true. Doing so, enumeration fields are always filled by the Workflow if the server response does not include them. 

Reviewer: ```suggestion
If, for any reason, your default value is semantically valid and you want the value to be included in the workflow model, you must set `kogito.grpc.enum.includeDefault` property to true. This way enumeration fields are always filled by the workflow if the server response does not include them. 
```
